homeassistant/components/withings/sensor.py

```diff
@@ -37,3 +41,8 @@ def native_value(self) -> None | str | int | float:
     def native_unit_of_measurement(self) -> str:
         """Return the unit of measurement of this entity, if any."""
         return self._attribute.unit_of_measurement
+
+    @property
+    def state_class(self) -> str:
+        """Return the state_class of this entity, if any."""
+        return SensorStateClass.MEASUREMENT
```

Comment:
This means ALL sensors are declared a measurement, but is that correct? Some entities represent totals too right?

Reply:
I believe they are all measurements.

Reply:
I dunno, for example, the distance sensor is not measurement IMHO, they are total increasing.

Reply:
Hmm, I can’t see a distance sensor in my version of the integration.

Reply:
That is possible, what is shown/exposed depends on what you are actually using.

Ref: <https://github.com/vangorra/python_withings_api/blob/master/withings_api/common.py#L190>

Reply:
Sorry, but not sure of the relevance of that code.

In the HA integration, the measurement types seems to be from https://github.com/home-assistant/core/blob/dev/homeassistant/components/withings/const.py#L18, which does not include distance.
The entities created appear to have to be on on the types in WITHINGS_ATTRIBUTES also defiend in https://github.com/home-assistant/core/blob/dev/homeassistant/components/withings/const.py#L123, which also does not have distance - and all seem to be measurements.